lib/routes/uraaka-joshi/uraaka-joshi-user.js

```diff
@@ -0,0 +1,19 @@
+const buildData = require('../../utils/common-config');
+
+module.exports = async (ctx) => {
+    const params = ctx.params;
+    const link = `https://www.uraaka-joshi.com/users/${params.id}`;
+    ctx.state.data = await buildData({
+        link,
+        url: link,
+        title: `$('.top-profile-card-name-link').text() + '@${params.id} - 裏垢女子まとめ'`,
+        item: {
+            item: '.content-main .stream .stream-item',
+            title: `$('.post-name').text() + '@${params.id} - 裏垢女子まとめ'`,
+            link: `https://www.uraaka-joshi.com/users/${params.id}`,
+            description: `$('.post .context').html()`,
```

Comment:
@SettingDust 
hi, it seems that we should handle the video item manually, not just parsing the html. The video tag misses the attribute "controls" and it won't show up at the rss reader ( e.g. feedly).